src/main/haskell/kore/src/Kore/IndexedModule/MetadataTools.hs

```diff
@@ -25,14 +28,17 @@ import Kore.IndexedModule.Resolvers
 -- access the metadata needed during the unification process.
 data MetadataTools level attributes = MetadataTools
     { symAttributes :: SymbolOrAlias level -> attributes
+    -- ^ get the attributes of a symbol or alias
     , sortAttributes :: Sort level -> attributes
-    , sortTools  :: SortTools level
+    -- ^ get the attributes of a sort
+    , symSorts :: SymSorts level
+    -- ^ get the signature of a symbol or alias
     , isSubsortOf :: Sort level -> Sort level -> Bool
+    {- ^ @isSubsortOf a b@ is true if sort @a@ is a subsort of sort @b@,
+       including when @a@ equals @b@. -}
     }
 
--- TODO: Rename this as `SortGetter` or something similar, `Tools` is
--- too general.
-type SortTools level = SymbolOrAlias level -> ApplicationSorts level
+type SymSorts level = SymbolOrAlias level -> ApplicationSorts level
```

Comment:
What do you suggest? I think `symbol` is a horrible way to refer to `SymbolOrAlias` because it misleadingly sounds like it's only `Symbol`. `sym` is at least obviously incomplete. `symbolOrAliasApplicationSorts` is not wrong, but it's just repeating the type signature in words.

Reply:
If you don't like `Symbol` (which, true, is suboptimal, though I think it's better than `Sym`), and `SymbolOrAlias` seems too long,  how about `Head`? I don't like it that much, but it seems that we've been using that terminology lately.

@traiansf What do you think about `Head`?

Reply:
`Head` seems reasonable to me.  But I would not want us to waste too much time on deciding this issue.